niCHARTPipelines/Structural.py

```diff
@@ -3,24 +3,59 @@
 import os,  shutil
 
 # from . import DeepMRSegInterface
-import nnUNetInterface
-import MaskImageInterface
-import ROIRelabelInterface
-import CalculateROIVolumeInterface
+from niCHARTPipelines import nnUNetInterface
+from niCHARTPipelines import MaskImageInterface
+from niCHARTPipelines import ROIRelabelInterface
+from niCHARTPipelines import CalculateROIVolumeInterface
 
-def run_structural_pipeline(inDir,DLICVmdl,DLMUSEmdl,outDir, MuseMappingFile,scanID,roiMappingsFile):
+def run_structural_pipeline(inDir,
+                            DLICVmdl_path,
+                            DLMUSEmdl_path,
+                            outDir, 
+                            MuseMappingFile,
+                            scanID,
+                            roiMappingsFile,
+                            nnUNet_raw_data_base=None,
+                            nnUNet_preprocessed=None,
+                            results_folder=None,
```

Comment:
Suggestion: better to rename this to 'model_folder' since it actually contains model. The Results_Folder name seems quite confusing. Internally it can still set the env variable os.environ['RESULTS_FOLDER']